TowerForge/TowerForge/Networking/RoomNetwork/GameRoom.swift

```diff
@@ -118,6 +120,18 @@ class GameRoom {
         }
     }
 
+    func updateGameMode(to mode: Mode) {
+        let roomStateRef = FirebaseDatabaseReference(.Rooms).child(roomName).child("gameMode")
+        roomStateRef.setValue(mode.rawValue) { error, _ in
+            if let error = error {
+                print("Error updating game mode: \(error.localizedDescription)")
```

Comment:
Can replace this with Logger.log(), cus we can universally turn off the debug messages later on via the constants class. The "self" argument will print the class filename to the console so it's easier to find where the messgae came from.
```suggestion
                Logger.log("Error updating game mode: \(error.localizedDescription)", self)
```